sycl/include/sycl/ext/oneapi/bindless_images.hpp

```diff
@@ -817,17 +825,36 @@ using OCLImageTyWrite =
 
 #define FETCH_UNSAMPLED_IMAGE(DataT, raw_handle, coords)                       \
   __invoke__ImageRead<DataT>(raw_handle, coords)
+
+#define FETCH_SAMPLED_IMAGE(DataT, raw_handle, coords)                         \
+  __invoke__ImageRead<DataT>(raw_handle, coords)
```

Comment:
Would this not still sample the data instead of fetching it from a sampled image?

Reply:
fetching a sampled image is using integer coordinates and backend implementation of the builtin is able to use load instead of sampling given that coordinate has integer type. Sampler can be ignored in this case.

Reply:
Makes sense.